imports/sql.go

```diff
@@ -166,12 +186,27 @@ func LoadFromSQL(ctx context.Context, stmt *sql.Stmt, options *SQLLoadOptions, a
 							return nil, fmt.Errorf("can't force string to bool. row: %d field: %s", row-1, fieldName)
 						}
 					case time.Time:
+						if database == MySQL {
+							t, err := time.Parse("2006-01-02 15:04:05", *val)
+							if err != nil {
+								// Assume unix timestamp
+								sec, err := strconv.ParseInt(*val, 10, 64)
+								if err != nil {
+									return nil, fmt.Errorf("can't force string to time.Time (%s). row: %d field: %s", *val, row-1, fieldName)
+								}
+								insertVals[fieldName] = time.Unix(sec, 0)
+							}
+							insertVals[fieldName] = t
```

Comment:
This is incorrect. You are writing here `insertVals[fieldName] = time.Unix(sec, 0)` and then overwriting again.

Reply:
pushed an update for this...